_docs-sources/pipelines/security/machine-users.md

```diff
@@ -82,7 +85,64 @@ The expiration of this token is up to you and the security posture of your organ
 ## Invite both machine users to Gruntwork
 Ensure both of these machine users are members of your team in **Gruntwork**’s GitHub Organization (See [instructions on inviting a user to your team](https://docs.gruntwork.io/developer-portal/invite-team#inviting-team-members) and [linking the user’s GitHub ID to Gruntwork](https://docs.gruntwork.io/developer-portal/link-github-id))
 
-## Repository Secrets
+### Configure secrets for GitHub Actions
```

Comment:
One more thing we might want to mention in these docs is that the repository access for these org level secrets will need to be updated any time an additional live or pipelines repo is created within their org. (Likewise, repo level secrets would need to be updated in that event as well). Perhaps we can call that out at the top of this section so it's clear that this isn't a one-time thing as more repos are created?

Reply:
Sounds good, I've added an additional `info` callout that mentions this